docroot/themes/contrib/civictheme/includes/table.inc

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @file
+ * Table theme alterations.
+ */
+
+define('CIVICTHEME_OPT_OUT', 'CivicThemeOptOut');
```

Comment:
we will introduce a more robust opt-out system later.

For now - can we please use 
```
define('CIVICTHEME_OPTOUT_TABLE', 'CivicThemeOptOutTable');
```